Joomla/Sniffs/Commenting/ClassCommentSniff.php

```diff
@@ -6,18 +6,17 @@
  * @copyright  Copyright (C) 2015 Open Source Matters, Inc. All rights reserved.
  * @license    http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License Version 2 or Later
  */
+namespace Joomla\Sniffs\Commenting;
 
-if (class_exists('Joomla_Sniffs_Commenting_FileCommentSniff', true) === false)
-{
-	throw new PHP_CodeSniffer_Exception('Class Joomla_Sniffs_Commenting_FileCommentSniff not found');
-}
-
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Util\Tokens;
 /**
```

Comment:
PHPCS removed their autoloader from composer which is the source of the issues
**solution** https://stackoverflow.com/questions/43707647/how-to-autoload-another-package-with-own-composer/43844960#43844960

**cause** squizlabs/PHP_CodeSniffer@566a0cd